src/member/mutations.test.ts

```diff
@@ -468,4 +473,31 @@ describe('Member Mutations', () => {
       );
     });
   });
+
+  describe('useExportMemberData', () => {
+    const mutation = mutations.useExportMemberData;
+    const response = 'Ok';
```

Comment:
You can use 'http-status-codes' 's ReasonPhrases  library
```suggestion
    const response = ReasonPhrases.OK;
```